src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -115,6 +115,11 @@ public void setStudent(Student target, Student editedStudent) {
         tutorsPet.setStudent(target, editedStudent);
     }
 
+    @Override
+    public void deleteAllStudents() {
+        tutorsPet.removeAllStudents();
```

Comment:
Perhaps it might be a good idea to have the same name for both methods (in `Model` and in `TutorsPet`)?